src/models/transfers.js

```diff
@@ -435,13 +437,18 @@ function * fulfillTransfer (transferId, fulfillmentUri) {
       }
     } catch (err) {
       // In case of a synchronization error, retry
-      if (Number(err.code) === 40001) {
-        retry(err)
+      if (Number(err.code) === 40001 || err.name === 'InsufficientFundsError') {
```

Comment:
turns out that `lib/db.js` from five-bells-ledger is no longer used by five-bells-ledger, opened an [issue](https://github.com/interledgerjs/five-bells-shared/issues/186) about that, and so there are no two retry layers. I'll still change which error comes through from src/lib/holds.js, though.